core/themes/basis/color/color.inc

```diff
@@ -0,0 +1,72 @@
+<?php
+
+$info = array(
+  // Available colors and color labels used in theme.
+  'fields' => array(
+    'header' => t('Header background'), // And active pager link.
+    'sitename' => t('Site name background'),
+    'slogan' => t('Slogan background'),
+    'titleslogan' => t('Site name and slogan'), // And main menu links.
+    'bg' => t('Main background'), // And submenu text, pager text, and drop down arrows
+    'footer' => t('Footer background'),
+    'text' => t('Text color'),
+    'link' => t('Link color'),
+    'hovermenu' => t('Hover menu link background'),
+    'border_thingy' => t('Border thingies'), //These are all over the place.
+    'form_focus_border' => t('Form element borders when selected'),
+    'action_item_background' => t('Action item background'),
```

Comment:
What's this mean? I don't understand it and I know the theme well...

Reply:
The action items are the 'view', 'edit', etc. links at the top pf pages.
![action_items](https://user-images.githubusercontent.com/1253691/39479611-9863b55c-4d1a-11e8-975f-def2f521390d.png)

Reply:
I'd just call it Tab background